internal/controller/promotions/promotions.go

```diff
@@ -274,31 +285,19 @@ func (r *reconciler) Reconcile(
 		return ctrl.Result{}, nil
 	}
 
-	if promo.Status.Phase == kargoapi.PromotionPhaseRunning {
-		// anything we've already marked Running, we allow it to continue to reconcile
-		logger.Debug("continuing Promotion")
-	} else {
-		// promo is Pending. Try to begin it.
-		if !r.pqs.tryBegin(ctx, promo) {
-			// It wasn't our turn. Mark this promo as Pending (if it wasn't already)
-			if promo.Status.Phase != kargoapi.PromotionPhasePending {
-				err = kubeclient.PatchStatus(ctx, r.kargoClient, promo, func(status *kargoapi.PromotionStatus) {
-					status.Phase = kargoapi.PromotionPhasePending
-				})
-				return ctrl.Result{}, err
-			}
-			return ctrl.Result{}, nil
+	if promo.Status.Phase == "" {
+		// Mark the Promotion as pending.
+		if promo.Status.Phase != kargoapi.PromotionPhasePending {
```

Comment:
Won't the second conditional _always_ be true if the first was?

Reply:
Yes, great catch.